Libraries/Cordio/platform/targets/maxim/max32655/sources/pal_timer.c

```diff
@@ -156,12 +156,6 @@ void PalTimerInit(PalTimerCompCback_t expCback)
   PAL_TIMER_CHECK(palTimerCb.state == PAL_TIMER_STATE_UNINIT);
   PAL_TIMER_CHECK(expCback != NULL);
 
-  /* TODO: Figure out priority for RISCV */
```

Comment:
FreeRTOS is very sensitive to interrupt priority. We set the priority at the application layer when we do the initialization.